data/World/Overworld.json

```diff
@@ -1570,12 +1773,30 @@
         "exits": {
             "DMC Upper Nearby": "True",
             "DMC Ladder Area Nearby": "True",
+            "DMC Pierre Platform": "
+                can_use(Goron_Tunic) and ((damage_multiplier != 'ohko' and damage_multiplier != 'quadruple') or 
+                (Fairy and not entrance_shuffle) or can_use(Nayrus_Love))",
```

Comment:
Access to Pierre Platform from DMT Upper Local does not actually require the Goron Tunic. It's fewer tunic requirements at worst, but DMC is actually pretty weird at the moment about where it requires the Goron Tunic. This route, when you're actually heading all the way to central, is the only one that hard requires it. Besides that, it only checks fewer_tunic_requirements to enter the temple, and nowhere else. It's pretty strange. Anyway, this shouldn't hard require tunic. You can use FTR if you want but it wouldn't be the most difficult thing at the moment that doesn't require any tunic. Crater rework is definitely on the agenda.

Also, the Fairy use is safe under entrance_shuffle. That's only a problem if you're planning to continue into Fire Temple, where you might encounter another required Fairy use with no way to replenish the one you lost on this fall.